.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI for Jupyter Notebooks
+
+on:
+  push:
+    branches:
+      - pm-41
```

Comment:
@adampingel if you merge this as it is- just do a hot fix, and change this line and line 9 (3 lower) from `pm-41` to `main`